README.md

```diff
@@ -40,3 +40,18 @@ may be referred to by other projects and scripts.
   The version specifies the exact version of the collection that appeared in that
   release of Ansible.  This file will be created by the ``antsibull-build single``
   command.
+
+## Adding a new collection
+
+### Next Ansible major release
+
+To add a collection to the next Ansible major release:
```

Comment:
```suggestion
To add a collection to the next Ansible major release that has not reached feature freeze:
```

Reply:
important point, thanks for spotting! committed